<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyMainWindowDemoClass</class>
 <widget class="QMainWindow" name="MyMainWindowDemoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyMainWindowDemo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPlainTextEdit" name="textContent"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>810</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="actionHelp"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>其他</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="separator"/>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionSettings"/>
   <addaction name="separator"/>
   <addaction name="actionHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="MyApplication.qrc">
     <normaloff>:/image/image/new_file.png</normaloff>:/image/image/new_file.png</iconset>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="MyApplication.qrc">
     <normaloff>:/image/image/open_file.png</normaloff>:/image/image/open_file.png</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="MyApplication.qrc">
     <normaloff>:/image/image/sava_file.png</normaloff>:/image/image/sava_file.png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="MyApplication.qrc">
     <normaloff>:/image/image/icon_settings.png</normaloff>:/image/image/icon_settings.png</iconset>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="MyApplication.qrc">
     <normaloff>:/image/image/help.png</normaloff>:/image/image/help.png</iconset>
   </property>
   <property name="text">
    <string>帮助</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MyApplication.qrc"/>
 </resources>
 <connections/>
</ui>
